lib/cocina/generator/schema_array.rb

```diff
@@ -9,7 +9,7 @@ def generate
       end
 
       def array_of_type
-        schema_doc.items.name || "Types::#{dry_datatype(schema_doc.items)}"
+        schema_doc.items.name || dry_datatype(schema_doc.items)
```

Comment:
The `Types::` prefix for built-in dry-types has moved below to avoid pre-pending it to custom/generated types, e.g., `Druid` or `SourceId`.

Reply:
👌🏻